### PR TITLE
Modify the response to match the spec (#48)

### PR DESCRIPTION
**Pull request by sdelatorrep**
Small modifications to the shape of the response to include the latest changes in the spec.

### CODE REVIEW

File: beacon/endpoints/rest/__init__.py [hunk 1]
```diff
@@ -77,14 +77,14 @@ class GVariantParametersBase(RequestParameters):
...
 
-        if values.targetIdReq is not None and (
-                len(values.start) > 0
-                or len(values.end) > 0
-                or values.referenceBases is not None
-                or values.alternateBases is not None
-                or values.assemblyId is not None
-                or values.referenceName is not None):
-            raise BeaconBadRequest("No other parameters are accepted when querying by Id")
+        # if values.targetIdReq is not None and (
```
Comment (silverdaz) [inline beacon/endpoints/rest/__init__.py:80]: Remove. Don't just comment.

File: beacon/endpoints/rest/__init__.py [hunk 2]
```diff
@@ -139,7 +139,7 @@ class GVariantsParameters(GVariantParametersBase):
                                   default='beacon-variant-annotation-v2.0.0-draft.2')
 
 
-def generic_handler(log_name, proxy, fetch_func, build_response_func):
+def generic_handler(log_name, by_entity_type, proxy, fetch_func, build_response_func):
```
Comment (silverdaz) [inline beacon/endpoints/rest/__init__.py:142]: Make all the entity_type integers and not strings. Use for example an enum.
Strings are interned in Python and you never know when they'll be copied. Moreover, strings comparison is slower than _one_ integer comparison.

**Review by silverdaz — CHANGES_REQUESTED** [review]
Refactoring needs to be done

File: beacon/endpoints/rest/__init__.py (hunk 2 above)
Comment (sdelatorrep) [reply beacon/endpoints/rest/__init__.py:133]: I've replaced it with `IntEnum`.

**Review by silverdaz — APPROVED** [review]
LGTM

**Pull request merged**